src/server.js

```diff
@@ -14,18 +14,15 @@ const {
     validateEnvironment,
 } = require('./environment');
 const { Firebase } = require('./firebase/firebase');
-const legacyServer = require('../listener/legacy-server');
-const legacyRegistrationServer = require('../legacy-registration/legacy-server');
 const legacyLogger = require('../listener/logs/logger');
-const { RequestHandler } = require('./core/request-handler');
 const { REQUEST_TYPE } = require('./const');
 const { Version } = require('./utility/version');
 
 // Raise AssertionError if environment variables are not set
 validateEnvironment(ENVIRONMENT);
 
 if (ENVIRONMENT.DATABASE_USE_SSL) {
```

Comment:
```suggestion
validateEnvironment(ENVIRONMENT);

// Validate SSL environment variables before importing the files that use them (legacyServer, etc.)
if (ENVIRONMENT.DATABASE_USE_SSL) {
```

Reply:
Good point. I added a comment above the first validate and added a bit more detail.